veps/sig-network/downwardapi-mac-address.md

```diff
@@ -0,0 +1,206 @@
+# VEP #127: Pass MAC address to network binding plugin sidecar
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [ ] (R) Target version is explicitly mentioned and approved
+- [ ] (R) Graduation criteria filled
+
+## Overview
+
+Pass the interface mac address through the network binding plugin
```

Comment:
Could you please elaborate on the source of this MAC address? Will it originate from the status annotation reported by Multus? (k8s.v1.cni.cncf.io/network-status)